src/Sulu/Bundle/SearchBundle/EventListener/ContentMapperSubscriber.php

```diff
@@ -69,7 +69,9 @@ public function onNodeSave(ContentNodeEvent $event)
     {
         $structure = $event->getStructure();
 
-        if ($structure->getNodeState() === Structure::STATE_PUBLISHED) {
+        // TODO: if the structure is the homepage, index it.
+        //       for some reason the content mapper never maps the state to published for a homepage
+        if ($structure->getPath() == '' || $structure->getNodeState() === Structure::STATE_PUBLISHED) {
```

Comment:
i think it should be initialized right because it cannot be changed in the ui! if it is not so we should change it in the `sulu:webspaces:init` command